python/ql/src/Security/CWE-117/LogInjection.ql

```diff
@@ -4,7 +4,8 @@
  *              insertion of forged log entries by a malicious user.
  * @kind path-problem
  * @problem.severity error
- * @precision high
+ * @security-severity 7.8
+ * @precision medium
```

Comment:
with this change, we need to update the change-note, since it talks about this query being part of the default pack, which it no longer will be. (but security-extended)

Reply:
Good point!